.github/workflows/lint.yaml

```diff
@@ -38,14 +38,6 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Prepare build environment
-        run: .github/workflows/prepare-build-env.sh
-
-      - name: Set up Go
-        uses: actions/setup-go@v3
-        with:
-          go-version: ${{ env.GO_VERSION }}
-
       - name: Go caches
         uses: actions/cache@v3
         with:
```

Comment:
I guess it doesn't use the cache then either, at least not like that. There's some kind of docker layer caching thing for github runners.

Reply:
The cache should be used. It will be mounted into the container.

Reply:
Right